src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -250,6 +250,18 @@ class AuthScreens extends React.Component {
                         return ConciergePage;
                     }}
                 />
+                <RootStack.Screen
+                    name={SCREENS.REPORT_ATTACHMENTS}
+                    options={{
+                        headerShown: false,
+                        presentation: 'transparentModal',
+                    }}
+                    getComponent={() => {
+                        const ReportAttachments = require('../../../pages/home/report/ReportAttachments').default;
+                        return ReportAttachments;
+                    }}
+                    listeners={modalScreenListeners}
+                />
```

Comment:
Utilizing `RootStack.Screen` proved to be the only viable solution for overlaying the modal over the opened chat. I tested alternative navigators such as `CENTRAL_PANE_NAVIGATOR`, `FULL_SCREEN_NAVIGATOR`, and `RIGHT_MODAL_NAVIGATOR`. However, these alternatives resulted in a blank screen beneath the modal. This undesired outcome was visible for at least half a second during the modal's closing transition.

#### Sample Behavior When Not Using `RootStack`

Please refer to the following link for a demonstration of this behavior without the use of `RootStack`:

[Behavior Without RootStack](https://github.com/Expensify/App/assets/12156624/7304bfc4-e893-46d5-99b8-333448cf098a)

#### Sample Behavior When Using `RootStack`

For comparison, the following link shows the preferred outcome achieved when utilizing `RootStack`:

[Behavior With RootStack](https://github.com/Expensify/App/assets/12156624/87b7af30-8eef-484c-b830-591c31e9ba17)